modules/core/src/v2/baseCoin.ts

```diff
@@ -347,6 +348,14 @@ export abstract class BaseCoin {
     return signMessage(message, bip32.fromBase58(key.prv), utxolib.networks.bitcoin);
   }
 
+  /**
+   * Decompose a raw transaction into useful information.
+   * @param options - coin-specific
+   */
+  explainTransaction(options: any): Promise<TransactionExplanation<any, string | number> | undefined> {
```

Comment:
Would `Record<string, any>` make sense for `options`? It looks like that should be compatible with eos.